qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDbConnector.groovy

```diff
@@ -67,8 +72,30 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
 
   @Override
   List<Customer> findCustomer(String firstName, String lastName) throws DatabaseQueryException {
-    throw new RuntimeException("Method not implemented.")
+    String sqlCondition = "WHERE firstName = ? AND lastName = ?"
+    String queryTemplate = CUSTOMER_SELECT_QUERY + " " + sqlCondition
+    List<Map> resultRows = new ArrayList()
+
+    Connection connection = connectionProvider.connect()
+    connection.withCloseable {
+      PreparedStatement preparedStatement = it.prepareStatement(queryTemplate)
+      preparedStatement.setString(1, firstName)
+      preparedStatement.setString(2, lastName)
+      ResultSet resultSet = preparedStatement.executeQuery()
+      while (resultSet.next()) {
+        resultRows.add(SqlExtensions.toRowResult(resultSet))
+      }
+    }
+    List<Customer> customerList = new ArrayList<>()
+    resultRows.forEach {Map row ->
+      AcademicTitle title = TITLE_FACTORY.getForString(row.academicTitle as String)
+      List<Affiliation> affiliations = fetchAffiliationsForPerson(row.id as int)
+      Customer customer = new Customer(row.firstName as String, row.lastName as String, title, row.eMailAddress as String, affiliations)
```

Comment:
```suggestion
      Customer customer = new Customer(row.firstName as String, row.lastName as String, title, row.eMailAddress as String, affiliations)
```
Keep in mind that this will have to be updated to a Customer Builder Pattern as soon as it is merged to data-model-lib